ui/src/submissions/conferences/schemas/constants.js

```diff
@@ -0,0 +1,248 @@
+import getValue from '../../common/getValue';
+
+export const countryOptions = [
```

Comment:
don't tell me you hard-coded a country list.
There must be a list CountryCode - Country somewhere.

Reply:
frontend doesn't have to be aware of country code, we need display only things there.

Reply:
I would be interested to hear how else we could display country select box?

Reply:
there must be a country list from another pull-down/autocomplete/whatever in this project.
Worst case: loop over that list to fill whatever you need here.

I just don't want to maintain several lists in case another former soviet union country pops up or they rename Macedonia.

Reply:
https://github.com/inspirehep/inspire-dojson/blob/702d2733fc6f1288ef4fb54687442fe332acb6a0/inspire_dojson/utils/geo.py

Reply:
- No we don't have another "pull-down/autocomplete/whatever" country list on the UI.
- Loop over what?
- Well, fortunately it's not often that a country pops up or renamed.
- Effort to share code between `dojson` and `inspirehep/ui` is way too much.